csharp/test/Microsoft.ML.OnnxRuntime.Tests.NetCoreApp/Microsoft.ML.OnnxRuntime.Tests.NetCoreApp.csproj

```diff
@@ -95,12 +103,12 @@
     <Compile Include="..\Microsoft.ML.OnnxRuntime.Tests.Common\Tensors\ArrayTensorExtensionsTests.cs">
       <Link>ArrayTensorExtensionsTests.cs</Link>
     </Compile>
-  </ItemGroup>
-
-  <ItemGroup Condition="'$(OnDeviceTraining)' == 'true'">
     <Compile Include="..\Microsoft.ML.OnnxRuntime.Tests.Common\TrainingTest.cs">
       <Link>TrainingTest.cs</Link>
     </Compile>
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TrainingEnabledNativeBuild)' == 'true'">
```

Comment:
I may misunderstand it :). does this mean, if enable_training is enabled, training_api related will also be built? so is there any need we have enable_training_on_device?

Reply:
This is the test project. Here we use "TrainingEnabledNativeBuild" to determine whether the native ort build includes training apis and use this to determine which tests to run.